Tools/vagrant/shellinit.sh

```diff
@@ -7,6 +7,9 @@ export PATH=$PATH:$HOME/jsbsim/build/src
 export BUILDLOGS=/tmp/buildlogs
 
 export APMROOT=/vagrant
+if [[ :$PATH: != *:"$HOME/.local/bin":* ]] && [ -d "$HOME/.local/bin" ] ; then
```

Comment:
Isn't `[[` a bash extension?

Reply:
right ! as we don't have shebang, it use bash by default on ubuntu, but I changed to sh version !